prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/CO/co-phd.yml

```diff
@@ -0,0 +1,224 @@
+      cliaForOutOfStateTesting: null
+      cliaForSender: {}
+      phoneNumberFormatting: "STANDARD"
+      suppressNonNPI: false
+      processingModeCode: null
+      replaceDiiWithOid: null
+      applyOTCDefault: false
+      useOrderingFacilityName: "STANDARD"
+      valueSetOverrides: {}
+      nameFormat: "STANDARD"
+      receivingOrganization: null
+      convertPositiveDateTimeOffsetToNegative: false
+      stripInvalidCharsRegex: null
+      convertDateTimesToReceiverLocalTime: false
+      useHighPrecisionHeaderDateTimeFormat: false
+      type: "HL7"
+      truncationConfig:
+        truncateHDNamespaceIds: true
+        truncateHl7Fields: []
+        customLengthHl7Fields: {}
+    jurisdictionalFilter:
+      - "(Bundle.entry.resource.ofType(Patient).address.state.exists() and Bundle.entry.resource.ofType(Patient).address.state\
+      \ = 'CO') or (Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state.exists()\
+      \ and Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state\
+      \ = 'CO')"
+    qualityFilter:
+        - "(%hl7MessageType.exists() and %hl7MessageType = 'R01')"
+        - "Bundle.entry.resource.ofType(DiagnosticReport).result.resolve().where(method.empty()\
+      \ or value.coding.code.empty()).count() = 0"
+        - "Bundle.entry.resource.ofType(DiagnosticReport).where( basedOn.resolve().extension('https://reportstream.cdc.gov/fhir/StructureDefinition/effective-date').empty()\
+      \ and specimen.resolve().collection.collected.empty() and effective.empty()\
+      \ and issued.empty() and result.resolve().effective.empty()).count() = 0"
+        - "Bundle.entry.resource.ofType(Patient).where(address.line[0].empty() and address.postalCode.empty()\
+      \ and telecom.where(system = 'phone').value.empty() and telecom.where(system\
+      \ = 'email').value.empty()).count() = 0"
+        - "Bundle.entry.resource.ofType(Patient).where(name.family.empty() or name.given.empty()\
+      \ or birthDate.empty()).count() = 0"
+        - "Bundle.entry.resource.ofType(ServiceRequest).performer.resolve().identifier.where(type.coding.code\
+      \ = 'CLIA').value.getIdType() = 'CLIA'"
+        - "Bundle.entry.resource.ofType(ServiceRequest).requester.resolve().organization.resolve().identifier.where(type.coding.code\
+      \ = 'CLIA').value.getIdType() = 'CLIA'"
+        - "Bundle.entry.resource.ofType(ServiceRequest).where(requester.resolve().practitioner.resolve().name.family.empty()).count()\
+      \ = 0"
+        - "Bundle.entry.resource.ofType(Specimen).where(type.empty()).count() = 0"
+        - "Bundle.identifier.value.empty().not()"
+    routingFilter:
+    processingModeFilter: []
+    reverseTheQualityFilter: true
+    conditionFilter: []
+    mappedConditionFilter: []
+    deidentify: false
+    deidentifiedValue: ""
+    timing:
+      operation: "MERGE"
+      numberPerDay: 4
```

Comment:
Do they only need us to send 4 reports per day?  Normally, I put 1440 (send every minute).  In other word, we send them as much as possible.

Reply:
These are their existing settings. I will leave like this for now and will check with them if they want that updated.